src/logic/safe/store/reducer/safe.ts

```diff
@@ -25,14 +25,11 @@ export const buildSafe = (storedSafe: SafeRecordProps): SafeRecordProps => {
   const names = storedSafe.owners.map((owner) => owner.name)
   const addresses = storedSafe.owners.map((owner) => checksumAddress(owner.address))
   const owners = buildOwnersFrom(Array.from(names), Array.from(addresses))
-  const activeTokens = Set(storedSafe.activeTokens)
-  const balances = Map(storedSafe.balances)
 
   return {
     ...storedSafe,
     owners,
-    balances,
-    activeTokens,
+    balances: [],
```

Comment:
We are now ignoring storedSafe balances here

Reply:
good catch, fixed!

Reply:
I think there is something better we can do there. Balances is a required prop from SafeRecordProps, so it has to have any value. We are spreading `storedSafe` at the top of the object, so balances value should be already there. As we are not doing anything with this property it should be safe just to simply delete it

Reply:
you're right :+1: